lib/ui/screen/home_page.dart

```diff
@@ -130,10 +152,22 @@ class _HomePageState extends State<HomePage> {
   Column _buildLoadingSpinner() {
```

Comment:
Change the name of this internal method, it is not a loading spinner it is the end text